packages/core/src/components/checkbox/checkbox.css

```diff
@@ -43,34 +38,115 @@
 
 .hds-checkbox,
 .hds-checkbox *,
-.hds-checkbox *::before,
-.hds-checkbox *::after {
+.hds-checkbox *:before,
+.hds-checkbox *:after {
   box-sizing: border-box;
 }
 
 .hds-checkbox:not(:first-of-type) {
   margin-top: var(--spacing-2-xs);
 }
 
+/* checkbox icon */
+.hds-checkbox .hds-checkbox__input:before {
```

Comment:
In core storybook the checboxes don't appear as selected. Could relate to the icon problem.

Reply:
Good point, need to check & fix

Reply:
@harriplappalainen this is related to css styles order in the css-file 🤦 if I change it according to lintering rules it breaks up 🤦 🤦

Reply:
Alphabetizing the rules already breaks this up 🤦